source/installation/deploy-minio-standalone.rst

```diff
@@ -1,63 +1,45 @@
-===============================
-Deploy MinIO in Standalone Mode
-===============================
+=====================================
+Deploy Single-Node Single-Drive MinIO
```

Comment:
I suggest "Deploy MinIO as Single-Node Single-Drive"
That better parallels the title for a distributed deployment page.

Reply:
Also, do we need to consider changing the file name?

Reply:
Yeah we can - I'll put a redirect up for a little while